### PR TITLE
Navigate to temporary feed page from landing

### LINKED ISSUE #38: As someone who lands on the splash page, I should be able to opt into the database
To support this, we will need:

**Database Model** -> `OptIn`, use whatever name you choose.
  * email

**Controller Changes** -> we will want to make a new opt in in the controller.

**View: should show the form** -> you may need to integrate with SimpleForm if this is the first form that we are making.

### PR DESCRIPTION
**Pull request by enico**
fixes #38 

Added Notice/Errors alert on Application Layout page
Added button that directs to "categories", but can be feed/all posts in future iteration, temp
Changed the background color from black to pattern so sign-in/sign-up pages can be seen

We were talking about css/colors in #44, I can close that one, and anybody can pick up the css/feel/colors if they want.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -5,6 +5,12 @@
                     <div class="col-md-8 col-md-offset-2">
                         <h1 class="splashtitle">Let's Pair</h1>
                         <p class="slogantext">A collaborative community of developers who love to code.
+                        <br />
+                        <br />
+                        <%= link_to categories_path do %>
+                        <button type="button" class="btn btn-primary btn-lg">Find a Session</button>
```
Comment (kenmazaika) [inline app/views/posts/index.html.erb]: Minor note here, but indentation should have the button nested one level deeper:

```
<%= link_to categories_path do %>
      <button type="button" class="btn btn-primary btn-lg">Find a Session</button>
 <% end %>
```

File: app/views/categories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h1>Place Holder</h1>
+
```
Comment (kenmazaika) [inline app/views/categories/index.html.erb]: We may want to add a test for this page.

**Comment by kenmazaika** [comment]
Looks great.  I made a couple of small notes, but I'm happy to merge the changes as it.

**Comment by cacauzen** [comment]
I can pick up look and feel if still available! @kenmazaika

**Comment by kenmazaika** [comment]
@cacauzen Sounds good.  Why don't you make a new GitHub issue for the specific changes to the appearance you want to make and we can discuss them as a group on GitHub.

**Pull request merged**